openedx/features/partners/views.py

```diff
@@ -18,6 +18,14 @@ def dashboard(request, slug):
     return views.dashboard(request, partner.slug)
 
 
+def performance_dashboard(request, slug):
+    partner = get_object_or_404(Partner, slug=slug)
+    if user_has_performance_access(request.user, partner):
+        partner_views = import_module_using_slug(partner.slug)
+        return partner_views.performance_dashboard(request, partner)
+    return HttpResponseForbidden()
```

Comment:
We should have a proper page for 403 response for better user experience. For now we can create a ticket for this purpose